src/pages/myGeotab/whatsNew.tsx

```diff
@@ -1,8 +1,3045 @@
+					<li>
+						<p><a href="/sdk/software/api/reference/#TextMessage">TextMessage</a> - 50,000 record limit</p>
+					</li>
+					<li>
+						<p><a href="/sdk/software/api/reference/#Trailer">Trailer</a> - 50,000 record limit</p>
+					</li>
+					<li>
+						<p><a href="/sdk/software/api/reference/#User">User</a> - 5,000 record limit</p>
+					</li>
+					<li>
+						<p><a href="/sdk/software/api/reference/#Zone">Zone</a> - 10,000 record limit</p>
+					</li>
+				</ul>
+			</li>
+		</ul>
+	</div>
+);
+
+//ToDo Update Object Links
+//ToDo Update Concepts Page Link
+const update1902: ReactNode = (
+	<div className="paragraph">
+		<ul>
+			<li>
+				<p><a href="/sdk/software/api/reference/#ApplicationVersionInformation">ApplicationVersionInformation</a>: Added beta support</p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#BinaryDataType">BinaryDataType</a>: Added <code className="small-code-sample">SoftwareVersionSection1</code>, <code className="small-code-sample">SoftwareVersionSection2</code>, <code className="small-code-sample">SoftwareVersionSection3</code></p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#DefectRemark">DefectRemark</a>: Added beta support</p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#DefectSeverity">DefectSeverity</a>: Added <code className="small-code-sample">Unregulated</code></p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#Device">Device</a>: Adding a device will now force the ActiveTo property to max date. Setting a device’s ActiveTo property to a future date, but not max date, will force the value to max date.</p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#DeviceType">DeviceType</a>: Added <code className="small-code-sample">GO9</code></p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#DiagnosticType">DiagnosticType</a>: Added <code className="small-code-sample">ProprietaryFault</code>, <code className="small-code-sample">LegacyFault</code></p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#DriverRegulation">DriverRegulation</a>: Added beta support</p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#DtcClass">DtcClass</a>: Added beta support</p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#DtcSeverity">DtcSeverity</a>: Added beta support</p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#DutyStatusAvailability">DutyStatusAvailability</a>: Added properties <code className="small-code-sample">CycleRest</code>, <code className="small-code-sample">DutySinceCycleRest</code>, <code className="small-code-sample">Is16HourExemptionAvailable</code>, <code className="small-code-sample">IsAdverseDrivingExemptionAvailable</code>, <code className="small-code-sample">IsOffDutyDeferralExemptionAvailable</code></p>
+			</li>
+			<li>
+				<p>DutyStatusAvailabilityDuration: Removed from documentionation, will be obsoleted in future</p>
```

Comment:
Typo from the current SDK page but might as well fix it

documentionation -> documentation